sansa-ml-spark/src/main/scala/net/sansa_stack/ml/spark/entity_resolution/Commons.scala

```diff
@@ -35,22 +39,22 @@ class EntityResolution_RDFData_CountVectorizer(spark: SparkSession, triplesRDD1:
   def run(): RDD[(String, String)] = {
```

Comment:
As I already mentioned it on my previous review, we should get rid of any main method into API. We could add unit-tests (by the way, I do not see any unit tests here, we usually should have at least some coverage with the functionality before we merge the PR) which cover some of the functionality and the main run we should consider moving it to the SANSA-Examples into [ml examples](https://github.com/SANSA-Stack/SANSA-Examples/tree/develop/sansa-examples-spark/src/main/scala/net/sansa_stack/examples/spark/ml).

Reply:
This method is required, so as to accomplish the 3-step procedure for our algorithm. As the output from each step becomes the input for the next, being a sequential process. We are including unit tests separately.